rest/attachment_test.go

```diff
@@ -2675,8 +2669,8 @@ func TestCBLRevposHandling(t *testing.T) {
 		btc := btcRunner.NewBlipTesterClientOptsWithRT(rt, &opts)
 		defer btc.Close()
 
-		doc1Version := btc.rt.PutDoc(doc1ID, `{}`)
-		doc2Version := btc.rt.PutDoc(doc2ID, `{}`)
+		doc1Version := btc.rt.PutDocDirectly(doc1ID, JsonToMap(t, `{}`))
+		doc2Version := btc.rt.PutDocDirectly(doc2ID, JsonToMap(t, `{}`))
```

Comment:
```suggestion
        startingBody := db.Body{"foo": "bar"}
		doc1Version := btc.rt.PutDocDirectly(doc1ID, db.Body{})
		doc2Version := btc.rt.PutDocDirectly(doc2ID, db.Body{})
```

you can drop the double `WaitForPendingChanges` too.